src/osd/PrimaryLogPG.cc

```diff
@@ -2344,6 +2344,17 @@ PrimaryLogPG::cache_result_t PrimaryLogPG::maybe_handle_manifest_detail(
     }
     return cache_result_t::HANDLED_PROXY;
   case object_manifest_t::TYPE_CHUNKED:
+    if (obc->obs.oi.size == 0) {
```

Comment:
size == 0 means that this object is flushed (only remote copy). So we need to promote.